docs/guides/universal-profile/edit-profile.md

```diff
@@ -16,7 +16,7 @@ This guide will teach you how to **customize our Universal Profile** programmati
 
 - adding a profile and cover picture to our Universal Profile,
 - editing our Universal Profile infos (e.g., description, badges, links),
-- see the updated profile details and images of our Universal Profile on the [l16.universalprofile.cloud](https://l16.universalprofile.cloud) website.
+- see the updated profile details and images of our Universal Profile on the [universalprofile.cloud](https://universalprofile.cloud/) website.
```

Comment:
Universal Profile . cloud is connected to L14, this change will make the tutorial not "valid" anymore as ppl won't be able to check on universalprofile.cloud